extension/lib/webcompat.js

```diff
@@ -85,20 +86,46 @@ class WebCompat {
   /**
    * @param {String} elementName -
    *                e.g. div, main, aside
+   * @param {Array} attributes -
+   *                e.g. [{ name: "dir", value: "ltr" }, ...]
    * @param {Array} browsers -
    *                e.g. [{ id: "firefox", name: "Firefox", version: "68" }, ...]
-   * @return {Object} if no issue found, return null.
+   * @return {Array} if there are no issues found, then the array is empty.
    */
-  getHTMLElementIssue(elementName, browsers) {
-    const database = this._webCompatData.html.elements;
-    const summary = this._getCompatSummary(browsers, database, elementName);
+  getHTMLElementIssues(elementName, attributes, browsers) {
+    const databaseElements = this._webCompatData.html.elements;
```

Comment:
Suggestion to rename this variable to `elementsDatabase`, or even just `elementsDB`.